pkgs/development/libraries/libptytty/default.nix

```
@@ -15,12 +21,24 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
+  cmakeFlags = lib.optional isStatic "-DBUILD_SHARED_LIBS=OFF"
+    ++ lib.optional (isCross || isStatic) "-DTTY_GID_SUPPORT=OFF"
+    # Musl lacks UTMP/WTMP built-in support
+    ++ lib.optionals isMusl [
+      "-DUTMP_SUPPORT=OFF"
+      "-DWTMP_SUPPORT=OFF"
+      "-DLASTLOG_SUPPORT=OFF"
+    ];
+
   meta = with lib; {
     description = "OS independent and secure pty/tty and utmp/wtmp/lastlog";
     homepage = "http://dist.schmorp.de/libptytty";
     maintainers = with maintainers; [ rnhmjoj ];
     platforms = platforms.unix;
```

```suggestion
    platforms = platforms.linux;
```
If this indeed only works on linux.
(The description says OS independent, what OS exactly?)

---

It appears to build on darwin before this change: https://hydra.nixos.org/eval/1790346?filter=libptytty&compare=1790331&full=